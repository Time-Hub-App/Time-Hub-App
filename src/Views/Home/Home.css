body {
  background-image: url(https://www.ppt-backgrounds.net/thumbs/light-blue-clipart-background.jpeg);
  background-size: cover;
  background-repeat: no-repeat;
  background-position: fixed;
  min-height: 100vw;
}
