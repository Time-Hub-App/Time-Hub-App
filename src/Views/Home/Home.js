import './Home.css';
import Header from '../../Components/Header/Header.js';
import Footer from '../../Components/Footer/Footer.js';
import logo from '../../images/TIMEHUB_LOGO.png';

export default function Home({ user, setUser }) {
  return (
    <>
      <Header user={user} setUser={setUser} home />
      <main>
        <img src={logo} />
        <p>
          Journaling is a very underrated activity that boosts your well-being in just a few minutes
          a day!
        </p>
        <p>
          A paper notebook and pen are fine for journaling, but TimeHub offers more! Jot your
          thoughts down or record what happened during the day for a simple way to manage stress,
          enhance creativity, increase happiness, and track your emotions.
        </p>
        <p>Click Log In to Sign up, or click Journals to start a new Journal</p>
      </main>
      <Footer />
    </>
  );
}
