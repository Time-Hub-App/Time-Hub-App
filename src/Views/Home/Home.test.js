import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import Home from './Home';

it('should render a home view component', () => {
  const { container } = render(
    <MemoryRouter>
      <Home />
    </MemoryRouter>
  );

  expect(container).toMatchSnapshot();
});
