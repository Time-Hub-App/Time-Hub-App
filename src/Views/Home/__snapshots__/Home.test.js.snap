// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a home view 1`] = `
<div>
  <header>
    <h3
      class="active"
    >
      Home
    </h3>
    <h3
      class=""
    >
      About
    </h3>
    <h3
      class=""
    >
      Log In
    </h3>
    <h3
      class=""
    >
      Journals
    </h3>
  </header>
  <main>
    <form>
      <h1>
        TimeHub
      </h1>
      <div>
        <h3>
          How To Use The App!
        </h3>
        <p>
          Instructions:
        </p>
      </div>
      <div>
        <input
          class="searchBar"
          type="search"
          value=""
        />
      </div>
      <div>
        <button
          class="save"
        >
          Save
        </button>
      </div>
    </form>
  </main>
  <footer>
    <a
      class="footer-link"
      href="/https://github.com/Time-Hub-App"
    >
      TimeHub GitHub
    </a>
  </footer>
</div>
`;

exports[`should render a home view component 1`] = `
<div>
  <header>
    <h3
      class="active"
    >
      Home
    </h3>
    <h3
      class=""
    >
      About
    </h3>
    <h3
      class=""
    >
      Log In
    </h3>
    <h3
      class=""
    >
      Journals
    </h3>
  </header>
  <main>
    <form>
      <h1>
        TimeHub
      </h1>
      <div>
        <h3>
          How To Use The App!
        </h3>
        <p>
          Instructions:
        </p>
      </div>
      <div>
        <input
          class="searchBar"
          type="search"
          value=""
        />
      </div>
      <div>
        <button
          class="save"
        >
          Save
        </button>
      </div>
    </form>
  </main>
  <footer>
    <a
      class="footer-link"
      href="/https://github.com/Time-Hub-App"
    >
      TimeHub GitHub
    </a>
  </footer>
</div>
`;
